notebooks/tutorial_calendar_shifter.ipynb

```diff
@@ -0,0 +1,1266 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    for run in range(n_runs - 1):
</pre><p>This seems like a less than optimal syntax. Can't we build a nice wrapper for the iteration and call the shifter from there?</p><p>E.g. calendar_shifter(base_calendar, stagger, n_shifts)?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AI4S2S/s2spy/pull/149/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>